lambdas/shared/t4_lambda_shared/utils.py

```diff
@@ -105,3 +105,30 @@ def buffer_s3response(s3response):
         raise IncompleteResultException("Error: Received an incomplete response from S3 Select.")
     response.seek(0)
     return response
+
+
+def call_s3_select(
+    s3_client: str,
+    bucket: str,
+    key: str,
+    sql_stmt: str
+) -> io.StringIO:
+    """
+    Call S3 Select to read only the logical keys from a
+    package manifest that match the desired folder path
+    prefix
+    """
+
+    print(sql_stmt)
```

Comment:
Fine but you really need to give this some context:
```suggestion
    print(f"utils.py: manifest_select: {sql_stmt})
```